config/default/jenkins-infra.yaml

```diff
@@ -453,11 +454,11 @@ jenkins:
               globalMatrix:
                 permissions:
                   - "Overall/Administer:admins"
-                  - "Overall/SystemRead:all"
-                  - "Overall/Read:all"
-                  - "Job/Read:all"
-                  - "Job/Build:all"
-                  - "Job/ExtendedRead:all"
+                  - "Overall/SystemRead:authenticated"
```

Comment:
what is the authenticated? Is it supposed to be a ldap group?

Reply:
No, it's a virtual group in Jenkins, which maps to any authenticated user through any of the defined realm. It's been added following a tip from Daniel in IRC, which make sense in this instance's case.